### PR TITLE
[DOC-2851] Doc: Update content of services overview and deployment ov…

### PR DESCRIPTION
**Pull request by deepakbolangady**
…erview. Updated docs for the Overview and Services dashboards in CG with new information about RBAC-based filtering of service instances. Tech reviewers: Sudarshan Purohit and Boopesh Shanmugam.

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* DOC-2851*: Doc: Update content of services overview and deployment overview

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/first-gen/firstgen-platform/fg-monitoring/main-and-services-dashboards.md [hunk 1]
```diff
@@ -10,9 +10,20 @@ helpdocs_is_published: true
 
 Harness Manager's Main Dashboard and Services Dashboard offer predefined views of your principal deployment data.
 
+
+:::info
+The following panels show only those service instances for which the user has `Read` permissions:
```
Comment (rashminandasahoo) [inline docs/first-gen/firstgen-platform/fg-monitoring/main-and-services-dashboards.md:15]: How about "The following panels display only service instances for which the user has `Read` permissions:"

File: docs/first-gen/firstgen-platform/fg-monitoring/main-and-services-dashboards.md [hunk 2]
```diff
@@ -10,9 +10,20 @@ helpdocs_is_published: true
...
+
+The required authorization information is obtained from the application permissions assigned to the user through their user group.
+
+Permissions-based filtering of service instances is behind the feature flag `SPG_SERVICES_OVERVIEW_RBAC`. To enable the feature flag, contact Harness Support.
+:::
+
 ### Main Dashboard
 
-The Main Dashboard foregrounds deployment and instance activity. It includes breakdowns by successful versus failed deployment, by Harness Service, and by other factors.
+The Main Dashboard foregrounds deployment and instance activity.  It includes breakdowns by successful versus failed deployment, by Harness Service, and by other factors.
```
Comment (aimurphy) [inline docs/first-gen/firstgen-platform/fg-monitoring/main-and-services-dashboards.md:26]:
```suggestion
The Main Dashboard foregrounds deployment and instance activity. It includes breakdowns by successful versus failed deployment, by Harness Service, and by other factors.
```

**Review by aimurphy — APPROVED** [review]
approved w one item - extra whitespace

**Review by aimurphy — APPROVED** [review]

File: docs/first-gen/firstgen-platform/fg-monitoring/main-and-services-dashboards.md (hunk 1 above)
Comment (SudarshanPurohit) [reply docs/first-gen/firstgen-platform/fg-monitoring/main-and-services-dashboards.md:15]: It isn't only Read permissions... the user could have either read or write permissions. Better to keep it as "user has access."

**Review by SudarshanPurohit — APPROVED** [review]

File: docs/first-gen/firstgen-platform/fg-monitoring/main-and-services-dashboards.md (hunk 1 above)
Comment (SudarshanPurohit) [reply docs/first-gen/firstgen-platform/fg-monitoring/main-and-services-dashboards.md:15]: Actually never mind. Read permissions sounds good, and more specific. Let's keep it as it is.

**Pull request merged**